.gitignore

```diff
@@ -265,3 +265,4 @@ artifacts/
 HelixPayload/
 /test/MUXControlsTestApp/MSTest/MUXControlsTestApp_Test
 /test/MUXControlsTestApp/TAEF/MUXControlsTestApp_Test
+*.csv
```

Comment:
I don't think it is true that we would never want to check in a csv file into this repo. Consider updating this to something more specific.

Reply:
Good point, I added this because the unreliable test report was showing up, but I can specify the default name of the report instead of this blanket ignore.